team-builder/src/App.js

```diff
@@ -1,21 +1,27 @@
 import React, {useState} from 'react';
-import {uuid} from 'uuid';
+import {v4} from 'uuid';
 import UserForm from './components/UserForm';
 import Users from './components/Users';
 
 
 function App() {
 
-  const [users, setUsers] = useState();
+  const [users, setUsers] = useState([
+    {
+      id: v4(),
+      name: "Sample User",
+      email: "a@be.com",
+      role: "Lab Minion"
+    }
+  ]);
 
   const addUser = user => {
     const newUser = {
-      id: uuid.v4(),
+      id: v4(),
```

Comment:
was using antiquated techniques for getting a UUID, just imported v4 instead, and used it directly